src/dlls/mscoree/mscoree.cpp

```diff
@@ -907,9 +907,23 @@ STDAPI GetCORSystemDirectoryInternal(__out_ecount_part_opt(cchBuffer, *pdwLength
     if (pBuffer == NULL)
         IfFailGo(E_POINTER);
 
+#ifdef CROSSGEN_COMPILE
```

Comment:
It sounds like that we are just not initializing the PAL the right way. @janvorli ideas?